disco_aws_automation/disco_storage.py

```diff
@@ -160,22 +160,19 @@ def create_snapshot_bdm(self, snapshot, iops):
         return device
 
     def configure_storage(self,
-                          hostclass,
                           ami_id=None,
                           extra_space=None,
                           extra_disk=None,
                           iops=None,
-                          ephemeral_disk_count=0,
-                          map_snapshot=True):
-        '''Alter block device to destroy the volume on termination and add any extra space'''
+                          ephemeral_disk_count=0):
+        ''' Cofigures block device mapping for root partition, extra disk if it's used,
+        and the ephemeral disks. Note that EBS volume mapping is no longer being created
+        here. The logic for attaching EBS volume has been moved to /etc/asiaq/init/ebs-start.sh '''
```

Comment:
i really suggest you to move /etc/asiaq/init/ebs-start.sh from asiaq_astro_config to asiaq under sample_configurations. ebs-start.sh should be like disco-associate-eni